macros/src/function_like/log.rs

```diff
@@ -76,6 +76,7 @@ fn necessary_features_for_level(level: Level, debug_assertions: bool) -> &'stati
         Level::Trace if debug_assertions => &["defmt-trace", "defmt-default"],
         Level::Debug if debug_assertions => &["defmt-debug", "defmt-trace", "defmt-default"],
 
+        Level::None => &["defmt-default"],
```

Comment:
Not sure if this is the right flag, decided not to introduce another flag here because it's kind of a default behaviour.

Reply:
Since `defmt::println!` should _always_ print, it should not require any feature. So you probably just need to return an empty slice (`&[]`).

Reply:
I had to specify `defmt-default`, otherwise the `log` sample program would not output the `println` statement.